RELEASE-NOTES.md

```diff
@@ -24,6 +24,9 @@
 ## Deprecations
 
 ## New additions
+* Added support for `native_app.package.post_deploy` scripts in project definition file.
```

Comment:
How do we want to communicate this in relation to `native_app.package.scripts`? Should we mark `scripts` as deprecated and encourage users to use this instead? Should we allow users to specify both?

Reply:
I think that's a good point. We can mark in as deprecated in the documentation. 
I am not sure if we should put deprecation warning in the code yet...

Reply:
What's our story around users that specify both? We should be very clear about the ordering.